### ISSUE TITLE
[Docs] Improve discoverability and accessibility of Discord community link

### ISSUE THREAD

**Issue by divega** [paraglider-project/paraglider#375]
The badge for the Paraglider Discord community on https://paragliderproject.io/ doesn't contain the word "Discord", only the Discord logo, and does not contain searchable text. 

We got at least one instance of customers not being able to find it: https://github.com/paraglider-project/paraglider/issues/244#issuecomment-2088645020. 

The accessibility is also bad, as it doesn't have a name, is not keyboard focusable and the role is "image", not "link". 

To see what I mean, contrast this to the GitHub star badge next to it.
